scripts/vagrant/multi/start_vagrant.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
Would this be cleaner if the start scripts were broken up by providers, and this script just called into the appropriate one?